test/unit/core.js

```diff
@@ -1190,6 +1190,18 @@ test("jQuery.each(Object,Function)", function() {
 	equal( i, document.styleSheets.length, "Iteration over document.styleSheets" );
 });
 
+test("jQuery.each/map(undefined/null,Function)", 1, function() {
```

Comment:
This landed and nobody noticed that it should have used `expect()` instead of the deprecated argument =/